packages/core/src/associations/belongs-to-many.ts

```diff
@@ -368,6 +368,8 @@ export class BelongsToManyAssociation<
       addMultiple: `add${plural}`,
       add: `add${singular}`,
       create: `create${singular}`,
+      createMultiple: `create${plural}`,
+      // TODO: add createMultiple association method for BelongsTo relationship
```

Comment:
```suggestion
      // TODO: add createMultiple association method for BelongsToMany relationship
```

Ideally this should be done in the same PR, as it now declares it as being available